src/app/shared/cookie-popup/_cookie-popup-theme.scss

```diff
@@ -16,6 +16,10 @@
         map.get(mat.$dark-theme-foreground-palette, secondary-text)
       );
       background: if($is-dark-theme, map.get(mat.$grey-palette, 50), #252525);
```

Comment:
It seems like this isn't working? the two colors are reversed? If it's a dark theme, it should use the black background, i.e. `#252525` and not the almost white `50` from the grey palette.

Maybe this would fix it without the need to change any accent/primary colors.

Reply:
Yes! is reverse right now!
but i like your idea, so with `color=primary` & popup background same as the theme we will have:

![Screenshot from 2021-06-07 22-49-14](https://user-images.githubusercontent.com/48172978/121070107-8e124e00-c7e3-11eb-8cbb-3b20e41c9d71.png)
![Screenshot from 2021-06-07 22-49-27](https://user-images.githubusercontent.com/48172978/121070138-979bb600-c7e3-11eb-8ca1-ad888705a86a.png)

looks good to you?
@Splaktar

Reply:
@crisbeto This LGTM. WDYT?

I don't like the idea of popping up a blinding white background dialog in dark mode. But I wanted to check if you had intended that?

Reply:
I was mimicking the color scheme from the Material snack bar. The only difference is that I made the background ~5% darker to meet the accessibility requirements.